src/Server/AllServers.ts

```diff
@@ -144,16 +149,27 @@ export function initForeignServers(homeComputer: Server): void {
       organizationName: metadata.organizationName,
     };
 
-    if (metadata.maxRamExponent !== undefined) {
-      serverParams.maxRam = Math.pow(2, toNumber(metadata.maxRamExponent));
-    }
-
     if (metadata.hackDifficulty) serverParams.hackDifficulty = toNumber(metadata.hackDifficulty);
     if (metadata.moneyAvailable) serverParams.moneyAvailable = toNumber(metadata.moneyAvailable);
     if (metadata.requiredHackingSkill) serverParams.requiredHackingSkill = toNumber(metadata.requiredHackingSkill);
     if (metadata.serverGrowth) serverParams.serverGrowth = toNumber(metadata.serverGrowth);
 
     const server = new Server(serverParams);
+
+    if (metadata.networkLayer) {
+      const maxCount = Math.floor(toNumber(metadata.networkLayer) / 2);
+      const minCount = Math.ceil(toNumber(metadata.networkLayer) / 4);
```

Comment:
For network layer 1 the "maxCount" is 0 since it's floor(1/2) but the minCount is 1 since it's ceil(1/4). maxCount should not be lower than minCount.